app.js

```diff
@@ -45,14 +42,19 @@ app.get(`/${api}`, async (req, res) => {
 });
 
 // GET /staff-fee-privilege/:id
-app.get(`/${api}/:id`, async (req, res) => {
+app.get('/staff-fee-privilege/:id', async (req, res) => {
   try {
     const { id } = req.params;
-    const result = await db.getStaffFeePrivilegesById(id);
-    if (_.isEmpty(result.data)) {
-      res.status(404).send(notFound('A staff fee privilege record with the specified ID was not found.'));
+    const [osuId, term] = id.split('-');
+    if (!osuId || !term) {
+      res.status(400).send(badRequest('ID format is not correct.'));
```

Comment:
Should we just return a 404 here instead? I'm not even sure we should be validating the format of the ID, unless it was a client generated ID. IMO, this resource asks for an ID which is some string. If no record matches that ID, we return a 404. Just my 2 cents.